assets/js/prosemirror/blocks.js

```diff
@@ -32,11 +36,14 @@ export default (options) => {
 
   const heading = [];
 
-  jsonOptions.map((option) => {
-    if (allowedHeading[option]) {
-      heading.push(allowedHeading[option]);
-    } else
-      map[option] = blocks[option];
+  blocksSelection.map((/** @type {Object} */ blockSelection) => {
```

Comment:
Do we really want the type in the comment like that? My tsserver recommends it too but it looks funny 🤷‍♂️

Reply:
yeah It's look a little bit weird IMO but it define what is the variable so I guess it's not bad to have